.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: php
+php:
+  - 7.4
+
+before_script:
+  - composer install --dev --no-interaction
+
+script:
+  - php vendor/bin/phpunit -c phpunit.xml
```

Comment:
I suggest to put all tests scripts necessary in a group scripts called `@test` including all tests.

An example [here](https://github.com/LyseonTech/dou-collector/blob/3f6e7ee0555f4730ef98f5b7dbd60623f8c1a703/composer.json#L34)